client/packages/system/src/Immunisation/api/operations.graphql

```diff
@@ -88,6 +88,16 @@ fragment VaccineCourse on VaccineCourseNode {
   isActive
   demographicIndicator {
     name
+    id
+  }
+  vaccineCourseItems {
```

Comment:
If you make these a fragments as well, it might work better in the code, and the node type has extra fields in it.